### PR TITLE
Rules and types

### PR DESCRIPTION
**Pull request by Astarsys**
Added rules folder
Added rule_types.md - there we can find a parameter list for every rule on the platform.
Optimized rule file.
Replaced binded to bound.

### CODE REVIEW

**Comment by Astarsys** [comment]
Ou, commits from the last pull request are here too.

File: docs/backend-api/resources/tracking/tracker/rules/rule_types.md [hunk 1]
```diff
@@ -0,0 +1,1332 @@
...
+
+# Rule types
+
+Rule types with all parameters to create. The rule availability depends on the device and rule integration for it.
+
+### Geofence entrance or exit
+
+A rule that triggers on entering/exiting geofences.
+
+#### parameters
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/rules/rule_types.md:14]: So much copy&paste... Extract common parameters to one table, plase.

**Review by fenik17 — CHANGES_REQUESTED** [review]

File: docs/backend-api/resources/tracking/tracker/rules/rule_types.md (hunk 1 above)
Comment (fenik17) [reply docs/backend-api/resources/tracking/tracker/rules/rule_types.md:14]: > Ou, commits from the last pull request are here too.

Just close prevous review

Comment (Astarsys) [reply docs/backend-api/resources/tracking/tracker/rules/rule_types.md:14]: will change the file and commit
closed another request

Comment (Astarsys) [reply docs/backend-api/resources/tracking/tracker/rules/rule_types.md]: commited

**Review by fenik17 — APPROVED** [review]
LGTM

**Pull request merged**